doc/devices/ahs_local.rst

```diff
@@ -0,0 +1,133 @@
+The local AHS device
+====================
+
+The local analog Hamiltonian simulation (AHS) device of the PennyLane-Braket plugin runs simulation on the local Braket SDK. This
+could be either utilizing the processors of your own PC, or those of a `Braket notebook instance <https://docs.aws.amazon.com/braket/latest/developerguide/braket-get-started-create-notebook.html>`_ hosted on AWS.
+
+This device is useful for small-scale simulations in which the time of sending a job to a remote service would add
+an unnecessary overhead. It can also be used for rapid prototyping before running a computation
+on a paid-for remote service.
+
+
+Usage
+~~~~~
+
+Connecting to the device will require installing the `Amazon Braket SDK <https://github.com/aws/amazon-braket-sdk-python#prerequisites>`_, which can be done via
+
+>>> pip install amazon-braket-sdk
+
```

Comment:
```suggestion
```
It's not clear to me that this needs to be mentioned, and it isn't for the local qubit simulator. 

Also, isn't `amazon-braket-sdk` a [dependency](https://github.com/aws/amazon-braket-pennylane-plugin-python/blob/605252299cb5fd7b8417b41b24802ac9f2c9a345/setup.py#L30) of the plugin?